x/superfluid/types/times.go

```diff
@@ -0,0 +1,8 @@
+package types
+
+import (
+	"time"
+)
+
+const Twoweeks = time.Hour * 24 * 14
```

Comment:
Lets rename to OsmosisMainnetUnbondingPeriod ?

Reply:
heh, I thought of this :)!

sure thing